### DISCUSSION TITLE
Decode list body

### DISCUSSION THREAD

**Discussion by MOsterhaus1** [fennelLabs/fennel-service-api#26] [Q&A]
A quick question about the `/v1/whiteflag/decode_list/` endpoint. Is the body of this endpoint a list of signal id's? When I call the endpoint `/v1/fennel/get_signals/` I got an signal with id = 3 in the response. However, when I call `/v1/whiteflag/decode_list/` with body `"signals": [3]` I get an `404 not found` response.

**Comment by Romulus10** [comment]
That endpoint definitely does take a list of signal IDs, so it looks like this one might be a bug. I'll get this fixed, thanks for letting us know about it!

**Reply by Romulus10** [reply]
@MOsterhaus1 I just shipped a patch to help out with this - let me know if you still have any problems.
